### DISCUSSION TITLE
Welcome to dns66 Discussions!

### DISCUSSION THREAD

**Discussion by julian-klode** [julian-klode/dns66#452] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

### Code of conduct
Please adhere to the [code of conduct](https://github.com/julian-klode/dns66/blob/main/CODE_OF_CONDUCT.md).

**Comment by GitAzrael** [comment]
Hey, im not sure if youre still doing updates to the app, but I was wondering if itd be possible to add a feature to prevent tampering with it when the VPN is active? Ive been wanting to use it to help out with an internet addiction and I really like the app bc of the amount of freedom you have with it. Just an idea, keep up the good work man

**Comment by jindivik** [comment]
App still working well on Poco X6 running android 14 / Hyperos. To prevent VPN disconnections, you'd also need to turn off volte /vowifi
